docs/examples/ratify-verify-azure-cmd.md

```diff
@@ -1,296 +1,361 @@
+
+# Connect to the AKS cluster:
+az aks get-credentials --resource-group ${GROUP_NAME} --name ${AKS_NAME}
+
+export AKS_OIDC_ISSUER="$(az aks show -n ${AKS_NAME} -g ${GROUP_NAME} --query "oidcIssuerProfile.issuerUrl" -otsv)"
+```
+
+6. Establish federated identity credential. On AZ CLI `${RATIFY_NAMESPACE}` is where you deploy Ratify:
+
+```bash
+az identity federated-credential create \
+  --name ratify-federated-credential \
+  --identity-name "${IDENTITY_NAME}" \
+  --resource-group "${GROUP_NAME}" \
+  --issuer "${AKS_OIDC_ISSUER}" \
+  --subject system:serviceaccount:"${RATIFY_NAMESPACE}":"gatekeeper-system"
+```
+
+7. Create an Azure Key Vault and set an access policy. If you have an AKV, you can skip this step.
+
+```bash
+az keyvault create \
+  --resource-group ${GROUP_NAME} \
+  --name ${KEYVAULT_NAME}
+```
+
+8. Set the environmental variable for Azure Key Vault URI.
+
+```bash
+export VAULT_URI=$(az keyvault show --name ${KEYVAULT_NAME} --resource-group ${GROUP_NAME} --query "properties.vaultUri" -otsv)
+```
+
+9. Import your own private key and certificates. You can import it on the portal as well.
+
+```bash
+az keyvault certificate import \
+  --vault-name ${KEYVAULT_NAME} \
+  -n <Certificate Name> \
+  -f /path/to/certificate
+```
+ 
+10. Configure policy for user-assigned managed identity:
+    
+```bash
+az keyvault set-policy --name ${KEYVAULT_NAME} \
+  --certificate-permissions get \
+  --object-id ${IDENTITY_OBJECT_ID}
+```
+
+## Deploy Gatekeeper and Ratify on AKS 
+
+1. Deploy Gatekeeper from helm chart:
+
+```bash
+helm repo add gatekeeper https://open-policy-agent.github.io/gatekeeper/charts
+
+helm install gatekeeper/gatekeeper  \
+  --name-template=gatekeeper \
+  --namespace gatekeeper-system --create-namespace \
+  --set enableExternalData=true \
+  --set validatingWebhookTimeoutSeconds=5 \
+  --set mutatingWebhookTimeoutSeconds=2
+```
+
+2. Deploy Ratify from helm chart:
+
+```bash
+helm clone https://github.com/deislabs/ratify.git
+
+helm install ratify ./charts/ratify --atomic \
+  --namespace ${RATIFY_NAMESPACE} \
+  --set-file provider.tls.crt=${CERT_DIR}/server.crt \
+  --set-file provider.tls.key=${CERT_DIR}/server.key \
+  --set provider.tls.cabundle="$(cat ${CERT_DIR}/ca.crt | base64 | tr -d '\n')" \
+  --set oras.authProviders.azureWorkloadIdentityEnabled=true \
+  --set azureWorkloadIdentity.clientId=${IDENTITY_CLIENT_ID} \
+  --set akvCertConfig.enabled=true \
+  --set akvCertConfig.vaultURI=${VAULT_URI} \
+  --set akvCertConfig.cert1Name=${NOTARY_PEM_NAME} \
+  --set akvCertConfig.tenantId=${TENANT_ID}
+```
+
+3. Enforce Gatekeeper policy to allow only signed images can be deployed:
+
+```bash
+kubectl apply -f https://deislabs.github.io/ratify/library/default/template.yaml
+kubectl apply -f https://deislabs.github.io/ratify/library/default/samples/constraint.yaml
+```
+
+4. This step above may take around 10 minutes to complete and registration status can be checked by running the following command:
```

Comment:
this step seems out of place as deploing ratify doens't take 10min, i wonder if the "10 min" wait is on the cluster creation?

Reply:
Updated according to your latest PR